pkg/reconciler/clusteringress/clusteringress.go

```diff
@@ -106,6 +111,25 @@ func (c *Reconciler) Init(ctx context.Context, cmw configmap.Watcher, impl *cont
 	configStore.WatchConfigs(cmw)
 	c.BaseIngressReconciler.ConfigStore = configStore
 
+	c.Logger.Info("Setting up StatusManager")
+	resyncIngressOnVirtualServiceReady := func(vs *v1alpha3.VirtualService) {
+		// Reconcile when a VirtualService becomes ready
+		impl.EnqueueLabelOfClusterScopedResource(networking.ClusterIngressLabelKey)(vs)
+	}
```

Comment:
Could this just be:
```
resyncIngressOnVirtualServiceReady := impl.EnqueueLabelOfClusterScopedResource(networking.ClusterIngressLabelKey)
```

Reply:
Unfortunately, EnqueueLabelOfClusterScopedResource is func(obj interface{}), resyncIngressOnVirtualServiceReady expects a func(*v1alpha3.VirtualService)